README.md

```diff
@@ -1,24 +1,39 @@
-# README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Micro Reddit
 
-Things you may want to cover:
+## Purpose
 
-* Ruby version
+> This is a project done with the aim of understanding how ActiveRecord works in rails.
```

Comment:
You could add a description of the models and associations present in the project.